### ISSUE TITLE
bug report: output appears in the lf's screen when scrolling past some ".psd" files

### ISSUE THREAD

**Issue by aabbccddeeffggj** [jstkdng/ueberzugpp#143]
Hi, there's a bug which output appears when scrolling through some of the ".psd" files I have (while using lf file manager).
Using the original ueberzug this problem doesn't happen.
![image](https://github.com/jstkdng/ueberzugpp/assets/133555950/f142ebdc-4db9-49d5-b1fb-339d1c97a9b6)

**Comment by jstkdng** [comment]
if you are using arch, switch the `libvips-git` package

**Comment by melecoaze** [comment]
I'm having a similar problem with any kind of file: [https://i.imgur.com/JbobkPy.png](url)

Using Fedora 38 by the way

**Comment by jstkdng** [comment]
hm, that's the first time I see those messages. Don't really have a solution for that.

**Comment by melecoaze** [comment]
> hm, that's the first time I see those messages. Don't really have a solution for that.

I thought it could be a lf issue but it happens in ranger also if I switch the previewer to ueberzug.

**Comment by jstkdng** [comment]
yeah, could be an issue with onetbb. I'd ask maintainer of the onetbb fedora package.

**Comment by melecoaze** [comment]
> yeah, could be an issue with onetbb. I'd ask maintainer of the onetbb fedora package.

I don't understand much about the subject, but apparently fedora is still stuck in the tbb version 2020.3 according to [this](https://bugzilla.redhat.com/show_bug.cgi?id=2036372).

Could that be the problem?

**Comment by Sneethe** [comment]
I have the same problem as @melecoaze. Have tried `2021.7.0`, `2021.9.0` and the latest `2021.11.0` of onetbb (also packaged as `tbb`).
And while I'm not at all familiar with ueberzugpp, shouldn't the --silent flag be blocking these messages?

**Comment by jstkdng** [comment]
the `--silent` flag only blocks output from standard error, those messages must come from standard output

**Comment by jstkdng** [comment]
The `--silent` flag only silences stderr messages, those logs must be on stdout which is required for the `sixel`, `kitty`, `iterm2` and `chafa` backends.

@Sneethe Could you give me an image of the issue?, I'm gonna try to research it.

**Comment by jstkdng** [comment]
closing due to no response
